testutils/mongodb.go

```diff
@@ -110,7 +114,7 @@ func backingMongoDBClient() (*mongo.Client, error) {
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
-	client, err := mongo.NewClient(options.Client().ApplyURI(mongoURI))
+	client, err := mongo.NewClient(baseOptions.ApplyURI(mongoURI))
```

Comment:
check that baseOptions is not nil? think this will panic otherwise?

Reply:
updated! this is all just testutils stuff so wasn't too worried but probably better to add it

Reply:
oh yeah, I guess I was thinking that if some test instantiated this with a nil client it would panic. but yeah maybe we aren't even exposing that code path.